.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+<!--
+Thanks for contributing!
+-->
+
+Some things to make sure you have completed:
+- [ ] Bump Spec version.
+- [ ] Update last modified date.
```

Comment:
These are often done together. I don't feel strongly about this but wanted to throw it out there in case it saves space.

```suggestion
- [ ] Bump spec version and last modified date.
```

Side note: both of these items may become irrelevant if [DRIVERS-2253](https://jira.mongodb.org/browse/DRIVERS-2253) moves forward.